python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -168,6 +174,10 @@ async def slew_object(
             Default is `SkyAuto`, the rotator is positioned with respect to the
             North axis and is automacally wrapped if outside the limit. See
             `RotType` for more options.
+        dra : `float`
```

Comment:
I don't understand that these are used for. Seems a weird way to do non-sidereal tracking....

Reply:
Resolved in private